### PR TITLE
FIX: Fix link to new codecov in readme

### PR DESCRIPTION
**Pull request by mgrover1**
Remove the old link to coveralls in the README

### CODE REVIEW

**Review by zssherman — APPROVED** [review]
Thanks!

**Comment by zssherman** [comment]
@mgrover1 Do we need to update to the codecov badge here:
.. image:: https://codecov.io/gh/ARM-DOE/ACT/branch/master/graph/badge.svg
 :target: https://codecov.io/gh/ARM-DOE/ACT

**Comment by mgrover1** [comment]
@zssherman I made that change

File: README.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Atmospheric data Community Toolkit (ACT)
 ========================================
 
-|AnacondaCloud| |Coveralls| |Build| |Docs|
+|AnacondaCloud| |Build| |Docs|
```
Comment (mgrover1) [inline README.rst:5]:
```suggestion
|AnacondaCloud| |CodeCovStatus| |Build| |Docs|
```

**Comment by mgrover1** [comment]
Alrighty - @zssherman this should be good to go.

**Comment by zssherman** [comment]
@mgrover1 Sounds good! Once tests pass, feel free to merge

**Pull request merged**